components/Event.tsx

```diff
@@ -69,17 +77,41 @@ const Event = ({
               {date}
             </div>
             <div className="2/3 mb-3">
-              <div className="text-sm font-medium  flex flex-wrap">
-                SHARE:
-                <a href="" className="text-blue  px-0 ">
-                  &nbsp;FACEBOOK&nbsp;
-                </a>
-                <a href="" className="text-lightBlueAccent px-0 ">
-                  TWITTER&nbsp;&nbsp;
-                </a>
-                <a href="" className="text-redAccent px-0 ">
-                  GOOGLE+
-                </a>
+              <div className="text-sm font-medium  flex flex-row flex-wrap justify-center items-center">
+                <p>SHARE: &nbsp;</p>
+                <FacebookShareButton
+                  title={title}
+                  url={'https://sliit-foss-rsvp.web.app/'}
+                  hashtag="#sliit #sliitfoss #fosslk"
+                >
+                  <FacebookIcon lightingColor="white" round={true} size={25}>
+                    {' '}
```

Comment:
Rm `{'    '}`

Reply:
what is wrong with this? if we use an element without content we can use it like this.

Reply:
please check other files. there also have these elements.

Reply:
If it is auto added, remove it otherwise keep it